projects/1.18/assets/decoration-delight/decoration_delight/lang/zh_cn.json

```
@@ -0,0 +1,79 @@
+{
+  "advancements.checkmate_advancement.descr": "获得一块黑白瓷砖",
+  "advancements.checkmate_advancement.title": "将军!",
+  "advancements.decoration_101_advancement.descr": "欢迎来到装饰乐事!",
+  "advancements.decoration_101_advancement.title": "装饰101",
+  "advancements.pizza_advancement.descr": "获得烹饪音乐唱片",
+  "advancements.pizza_advancement.title": "烹饪之声",
+  "advancements.sink_advancement.descr": "使用洗碗槽.",
+  "advancements.sink_advancement.title": "水花四溅",
+  "advancements.trash_bin_advancement.descr": "获得垃圾桶",
+  "advancements.trash_bin_advancement.title": "果皮啊，垃圾啊，一定要丢进垃圾桶里面",
+  "block.decoration_delight.acacia_counter": "金合欢木柜台",
+  "block.decoration_delight.acacia_counter_bend": "金合欢木柜台转角",
+  "block.decoration_delight.acacia_counter_corner": "金合欢木柜台角落",
+  "block.decoration_delight.acacia_countertop": "金合欢木台面",
+  "block.decoration_delight.acacia_mosaic": "金合欢马赛克",
+  "block.decoration_delight.acacia_stool": "金合欢木凳",
+  "block.decoration_delight.andesite_tiles": "安山岩瓷砖",
```

```suggestion
  "block.decoration_delight.andesite_tiles": "安山岩瓦",
```
见原版`深板岩瓦`/`Deepslate Tiles`

---

用于厨房地板的装饰方块，此处用瓷砖可能更合适